index.js

```diff
@@ -92,11 +92,12 @@ const staticify = (root, options) => {
 
         const fileName = path.basename(p);
         const fileNameParts = fileName.split('.');
+        const fileNameHash = fileNameParts[fileNameParts.length - 2];
```

Comment:
Not related to this PR, but since I noticed it here, how about saving `fileNameParts.length` to a variable and reusing it?

Reply:
I'd happily do that, I didn't want to go too far with your module and interfere with how you do things, I'll update now.